bookwyrm/tests/views/test_rss_feed.py

```diff
@@ -127,3 +127,26 @@ def test_rss_quotation_only(self, *_):
         self.assertEqual(result.status_code, 200)
 
         self.assertIn(b"a sickening sense", result.content)
+
+    def test_rss_shelf(self, *_):
+        """load the rss feed of a shelf"""
+        with patch("bookwyrm.models.activitypub_mixin.broadcast_task.apply_async"):
```

Comment:
You need to add another mock here for `bookwyrm.activitystreams.add_book_statuses_task`:
```python
with patch("bookwyrm.models.activitypub_mixin.broadcast_task.apply_async"), patch(
"bookwyrm.activitystreams.add_book_statuses_task.delay"
)
```